src/main/kotlin/dev/arbjerg/ukulele/command/HelpCommand.kt

```diff
@@ -14,8 +14,22 @@ class HelpCommand : Command("help") {
 
     override fun HelpContext.provideHelp() {
         addUsage("")
-        addDescription("Displays general help. (unfinished)") // TODO
+        addDescription("Displays general help.")
         addUsage("<command>")
         addDescription("Displays help about a specific command.")
+        addDescription("")
+        addDescription("")
+        addDescription("Here are the currently available commands for Ukulele:")
+        addDescription("nowplaying / np")
+        addDescription("pause")
+        addDescription("play / p")
+        addDescription("prefix")
+        addDescription("queue / q / list / l")
+        addDescription("repeat / r / loop")
+        addDescription("resume")
+        addDescription("say")
+        addDescription("skip / s")
+        addDescription("stop")
+        addDescription("volume / v")
```

Comment:
Here's where I listed the available commands.  A more elegant solution would be to walk those type of components but this will be helpful for now.  I realized it was missing when I was trying to find the commands as a noob.